addons/account/models/account_move.py

```diff
@@ -528,6 +532,17 @@ def _sequence_fixed_regex(self):
         states={'draft': [('readonly', False)]},
         help='Defines the smallest coinage of the currency that can be used to pay by cash.',
     )
+    invoice_pdf_report_id = fields.Many2one(
+        comodel_name='ir.attachment',
+        string="PDF Attachment",
+        compute=lambda self: self._compute_linked_attachement_id('invoice_pdf_report_id', 'invoice_pdf_report_file'),
+        depends=['invoice_pdf_report_file']
+    )
+    invoice_pdf_report_file = fields.Binary(
+        attachment=True,
```

Comment:
```suggestion
```

Reply:
I know, but I think it is good to insist on that one because we are extremely dependent on that behavior